validation/schema.json

```diff
@@ -296,11 +296,55 @@
     "modified"
   ],
   "$defs": {
-    "ecosystem": {
```

Comment:
I wonder if changing this ecosystem name will break anyone's workflow.

Reply:
I thought about that, but figured the likelihood anyone is depending on the name specifically right now is pretty low especially given no one seems to have caught how out of date this field actually is 😅

(the name should only matter if you're extending the schema in some way - if you're just using it for validating JSON files then at most it should result in a slightly different error message which shouldn't be depended on anyway)